packages/api/amplify_api/test/web_socket/web_socket_bloc_test.dart

```diff
@@ -17,25 +17,32 @@ import 'dart:convert';
 
 import 'package:amplify_api/src/graphql/web_socket/blocs/web_socket_bloc.dart';
 import 'package:amplify_api/src/graphql/web_socket/state/web_socket_state.dart';
+import 'package:amplify_api/src/graphql/web_socket/types/connectivity_status.dart';
 import 'package:amplify_api/src/graphql/web_socket/types/web_socket_types.dart';
-import 'package:amplify_flutter/amplify_flutter.dart';
-import 'package:connectivity_plus/connectivity_plus.dart';
-import 'package:connectivity_plus_platform_interface/connectivity_plus_platform_interface.dart';
+import 'package:amplify_core/amplify_core.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 import '../util.dart';
 
 const mockConnectionAck =
     ConnectionAckMessageEvent(ConnectionAckMessagePayload(300000));
 
+late StreamController<ConnectivityStatus> mockNetworkStreamController;
+
+class MockConnectivity extends ConnectivityInterface {
+  const MockConnectivity();
+
+  @override
+  Stream<ConnectivityStatus> get onConnectivityChanged =>
+      mockNetworkStreamController.stream;
+}
+
```

Comment:
Let's move this to the utils file and we can reuse in the other tests when `connectivityStreamCreator` becomes non-null.

Reply:
moved it to util, but not used anywhere else, yet